modules/prebidServerBidAdapter/ortbConverter.js

```diff
@@ -101,6 +101,7 @@ const PBS_CONVERTER = ortbConverter({
     }
 
     const bidResponse = buildBidResponse(bid, context);
+    bidResponse.rendererurl = bid.ext.rendererurl;
```

Comment:
I've opened the server-side issue https://github.com/prebid/prebid-server/issues/3085 to discuss the location of this field. Am proposing seatbid[].bid[].ext.prebid.meta.rendererurl

Reply:
Done in PBS-Go 0.274.0 and PBS-Java 2.2